task3/simplyTable.js

```diff
@@ -0,0 +1,18 @@
+let mybtn = document.getElementById('btn')
+
+mybtn.onclick = function () {
+    setTimeout(function () {
+        const myStatus = document.querySelectorAll('.pending')
```

Comment:
What is `myStatus` variable? Out of code, it looks like string or object, but here is array. Maybe it's better to use `pendingStatuses` name?